### DISCUSSION TITLE
how linkedIn inMail works, how the receipient/users don't benefit and face spam, and how we can reward users when recruiters want to talk to them

### DISCUSSION THREAD

**Discussion by etherinsight** [Developer-DAO/DAO-job-board#142] [Ideas]
Context: we started this discussion on Twitter

https://twitter.com/EtherInsight/status/1468014281492938758
https://twitter.com/EtherInsight/status/1468014281492938758/retweets/with_comments

## Problem
Linkedin users get a lots of inMail which costs up to $10 per message. Companies and recruiters pay this to Linkedin to be able to reach out to users. It's gated and I'm personally not a huge fan of how LinkedIn has this setup. 

## Solution 
On Web3 job board/ professional network, when recruiters want to get in top of your inbox they pay up just like on LinkedIn but instead of the money going only to LinkedIn, it can be split between the recipient and the platform( maybe 10-20% for operational fees) 

## How to make relationships balanced between recruiters and developers/professionals

The reason why recruiters reach out to professionals over inMail is that recruiters like the user's bio/profile/resume on Linkedin. So someone could exploit this and inflate their job history and credentials. For this we need to ensure that whatever credentials is shown on profile is verified on blockchain. This means the recruiters will reach out to only real profiles and not waste money on fake profiles farming for the web3Mail(inMail) money/reward.  This way we keep things fair for both the job seekers and recruiters.

**Comment by carlomigueldy** [comment]
This is a very great idea, thanks for putting this up @etherinsight 

How do we verify the credentials on the smart contract / blockchain? Can you provide some high level steps if you already got something in mind

**Comment by Marzel7** [comment]
Really like the sound of this proof of competence model as it benefits both recruiter and user.

**Comment by etherinsight** [comment]
@carlomigueldy here are some ideas for credentials verification on the blockchain

- If one says they're NFT Traders : use blockchain to calculate activity, profit % / return on investment. 
- If one says they have defi knowledge: check defi activities( staking, liquidity mining, token trading) , check for badges from apps like Rabithole 
- if one says they're solidity devs, check badge from Buildspace, check if they have deployed contracts
- Some initiatives like https://digitalcredentials.mit.edu/ has options for students to store their credentials on the blockchain

Also, if we want to move towards this model of on-chain credential verification, we should probably create a verification platform where an organization can sign that a certain wallet/identity has earned some credentials.

**Reply by carlomigueldy** [reply]
Love it bro! Do you think we'd use an Oracle for these kind of computations? Or just a serverless function like AWS Lambda

This one is interesting and would love to explore more how we can implement it well

**Reply by etherinsight** [reply]
we can use Serverless functions like Lambda to verify the accomplishment(NFT trades, contract deployment etc) and then let users to mint their credentials on chain as an NFT via our credential verification platform. This way users can reuse the credentials elsewhere if they want. 

For existing accomplishments like Rabithole.gg badges, we just need to make sure that the badges/NFTs given by Rabithole is with the original receiver.

**Comment by GrowthHacker2050** [comment]
Good Stuff, great insights and info!
